ViAn/GUI/projectwidget.cpp

```diff
@@ -156,6 +161,62 @@ void ProjectWidget::add_basic_analysis(VideoProject* vid_proj, BasicAnalysis* ta
     vid_item->setExpanded(true);
 }
 
+/**
+ * @brief ProjectWidget::add_frames_to_tag
+ * Create the TagFrameItems from all the frames. Used when opening a project
+ * @param item
+ */
+void ProjectWidget::add_frames_to_tag(TreeItem* item) {
```

Comment:
Minor detail but we should change the function name to `add_frames_to_tag_item` so it is clear that the frames are added to a tree item and not a tag object